### DISCUSSION TITLE
https://demonkingswarn.is-a.dev/blog/2024/03/17/emulation-aint-evil

### DISCUSSION THREAD

**Discussion by giscus** [DemonKingSwarn/blog#4] [General]
# https://demonkingswarn.is-a.dev/blog/2024/03/17/emulation-aint-evil

https://demonkingswarn.is-a.dev/blog/2024/03/17/emulation-aint-evil

**Comment by RaghavGohil** [comment]
Good take. I agree with most of the stuff that's mentioned here but Nintendo is known for its abbhorent takedowns •_•
